### PR TITLE
Companion for #10324 - update frame-benchmarking/runtime-benchmarks

### PR DESCRIPTION
**Pull request by zjb0807**
skip check-dependent-cumulus

### CODE REVIEW

File: runtime/rococo/Cargo.toml [hunk 1]
```diff
@@ -176,7 +176,7 @@ runtime-benchmarks = [
 	"pallet-xcm/runtime-benchmarks",
 	"xcm-builder/runtime-benchmarks",
 	"pallet-multisig/runtime-benchmarks",
-	"frame-benchmarking",
+	"frame-benchmarking/runtime-benchmarks",
```
Comment (Jaysonmaldx) [inline runtime/rococo/Cargo.toml:179]: Thanks

Comment (zjb0807) [reply runtime/rococo/Cargo.toml:179]: Do I need to make any changes?

Comment (KiChjang) [reply runtime/rococo/Cargo.toml:179]: Known spam account, ignore.

**Review by bkchr — APPROVED** [review]

**Review by svyatonik — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Comment by bkchr** [comment]
bot merge

**Comment by bkchr** [comment]
bot merge

**Pull request merged**